source/features/vertical-front-matter.tsx

```diff
@@ -0,0 +1,37 @@
+import './vertical-front-matter.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const table = select('[data-table-type="yaml-metadata"]');
+	if (!table) {
+		return;
+	}
+
+	const thead = select('thead', table)!;
+	const tbody = select('tbody', table)!;
+	while (select.exists('th', thead)) {
+		const th = select('th', thead)!;
+		th.classList.add('rgh-front-matter-name');
+		tbody.append(
+			<tr>
+				{th}
+				{select('td', tbody)!}
+			</tr>
+		);
+	}
```

Comment:
Can you find a proper transpose algorithm? There’s a lot of `select`ion here

Reply:
> There’s a lot of `select`ion here.

Can I just use `tr.children[0]`? So there is no selection.

Reply:
`thead.children[index]` `tbody.children[index]` 

or use [`select.all`](https://github.com/fregante/select-dom#selectallselector-baseelements--document) to get 2 arrays back:

```javascript
const thead = select.all('thead tr th', table)!;
const tbody = select.all('tbody', table)!;
```

and then `thead[index]`, `tbody[index]`